docs/application/dotnet/guides/nui/nui-components/Pagination.md

```diff
@@ -0,0 +1,163 @@
+
+2. Set the Pagination property:
+
+    ```cs
+    var indicatorImageUrlStyle = new PaginationStyle()
+    {
+        IndicatorImageUrl = new Selector<string>
+        {
+            Normal = "pagination_ic_nor.png",
+            Selected = "pagination_ic_sel.png"
+        }
+    };
+    pagination.ApplyStyle(indicatorImageUrlStyle);
+    pagination.IndicatorSize = new Size(26, 26);
+    pagination.IndicatorSpacing = 8;
+    pagination.Name = "Pagination1";
+    pagination.Size = new Size(300, 50);
+    pagination.BackgroundColor = new Color(1.0f, 1.0f, 1.0f, 0.6f);
+    pagination.IndicatorCount = 5;
+    pagination.SelectedIndex = 1;
+    root.Add(pagination);
+    ```
+
+Following output is generated when the Pagination is created using property:
+
+![Pagination1](./media/Pagination1.PNG)
+
+## Create with style
+
+To create a Pagination using style, follow these steps:
+
+1. Create a style for Pagination:
+
+    ```cs
+    PaginationStyle style = new PaginationStyle()
+    {
+        IndicatorSize = new Size(15, 15),
+        IndicatorSpacing = 20,
+        IndicatorImageUrl = new Selector<string>
+        {
+            Normal = "pagination_ic_nor.png",
+            Selected = "pagination_ic_sel.png"
+        },
+        Name = "Pagination2",
+        Size = new Size(300, 50),
+        BackgroundColor = new Color(1.0f, 1.0f, 1.0f, 0.6f),
+    };
+    ```
+
+2. Use the style to create a Pagination and add it to parent:
+
+    ```cs
+    pagination = new Pagination(style);
+    pagination.IndicatorCount = 5;
+    pagination.SelectedIndex = 2;
+    root.Add(pagination);
+    ```
+
+Following output is generated when the Pagination is created using style:
+
+![Pagination2](./media/Pagination2.PNG)
+
+## Create with defined styles
+
+You can define a style based on the user experience (UX) and then use this style to create a Pagination.
+
+1. Define a custom style:
+
+    ```cs
+    internal class CustomPaginationStyle : StyleBase
+    {
+        protected override ViewStyle GetViewStyle()
+        {
+            PaginationStyle style = new PaginationStyle
+            {
+                IndicatorSize = new Size(15, 15),
+                IndicatorSpacing = 20,
+                IndicatorImageUrl = new Selector<string>
+                {
+                    Normal = "pagination_ic_nor.png",
+                    Selected = "pagination_ic_sel.png"
+                },
+                Name = "Pagination2",
+                Size = new Size(300, 50),
+                BackgroundColor = new Color(1.0f, 1.0f, 1.0f, 0.6f),
+            };
+            return style;
+        }
+    }
+    ```
+
+2. Register your custom style:
+
+    ```cs
+    StyleManager.Instance.RegisterStyle("CustomPagination", null, typeof(YourNameSpace.CustomPaginationStyle));
+    ```
+
+3. Use your custom style to create a Pagination instance:
+
+    ```cs
+    pagination = new Pagination("CustomPagination");
+    pagination.IndicatorCount = 5;
+    pagination.SelectedIndex = 2;
```

Comment:
Question. It is said 2nd is selected.

However in the image Pagination2.png, the 3rd is selected.
Would you please check this?

Reply:
The index begins with 0. If the total number is 5, the index will be 0,1,2,3,4
So "SelectedIndex = 2" means 3rd

Reply:
Thanks for the answer. If so, would you please check the above indexes?

line 35. 
    pagination.SelectedIndex = 1;
line 70.
    pagination.SelectedIndex = 2;

According to your description, they should be each 0 and 1.

Reply:
Thanks for the answer. I confused the image layout and the sample codes so attached wrong comment, and removed it from here.